tmt/steps/prepare/feature/epel-disable.yaml

```diff
@@ -10,36 +12,48 @@
 
         # Disable repos of 'epel' if package 'epel-release' is installed
         - name: Detect package 'epel-release' is installed
-          command: rpm -q epel-release
+          ansible.builtin.command: rpm -q epel-release # noqa: command-instead-of-module
           register: result
-          ignore_errors: True
+          ignore_errors: true
+          changed_when: false
 
         - name: Disable repos of 'epel'
-          shell: yum-config-manager --disable epel epel-debuginfo epel-source
           when: result.rc == 0
+          ansible.builtin.command: yum-config-manager --disable epel epel-debuginfo epel-source
+          register: output
+          changed_when: output.rc != 0
 
-    # RHEL N or CentOS Stream N where N >= 8
-    - when: ansible_distribution_major_version|int >= 8
+    # RHEL 8+ or CentOS Stream 8+
+    - name: Disable repos of 'epel' on RHEL 8+ or CentOS Stream 8+
+      when: ansible_distribution_major_version | int >= 8
       block:
         - name: Install 'dnf config-manager'
-          shell: dnf -y install 'dnf-command(config-manager)'
+          ansible.builtin.command: dnf -y install 'dnf-command(config-manager)'
+          register: output
+          changed_when: output.rc != 0
 
         # Disable repos of 'epel' if package 'epel-release' is installed
         - name: Detect package 'epel-release' is installed
-          command: rpm -q epel-release
+          ansible.builtin.command: rpm -q epel-release # noqa: command-instead-of-module
           register: result
-          ignore_errors: True
+          ignore_errors: true
+          changed_when: false
```

Comment:
Is this check necessary? I was thinking a bit about the exact meaning of the `epel` feature and I would like to suggest this:

 * `epel: enabled` ... epel repositories are configured and enabled
 * `epel: disabled` ... epel repositories are configured and disabled

In other words, always make sure the repo files are present. This would allow tests easily enable `epel` only when needed.

Reply:
> Is this check necessary? 

This is because of such a case of disabling `epel` repo in the following:
* package epel-release is not installed by user, we should do nothing.

> In other words, always make sure the repo files are present. This would allow tests easily enable epel only when needed.

This idea makes things easier and we can leverage [ansible.builtin.yum module](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/yum_module.html) which is suggesed by @martinhoyer to enable/disable `epel` repos.  @happz, what do you think?

Reply:
Apologies for not looking properly into yum module before mentioning it. It's not applicable for this, as described in the comment below. Also, dnf module should probably be preferred at this point.

Reply:
It doesn't matter, @martinhoyer. And it's not harmless but very helpful for us to determine the final solution by further understanding module [ansible.builtin.yum](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/yum_module.html), thank you :-)